### PR TITLE
HDDS-2522. Fix TestSecureOzoneCluster

### PR DESCRIPTION
**Pull request by adoroszlai**
## What changes were proposed in this pull request?

Fix `TestSecureOzoneCluster`, failing because it used the wrong principal for SCM.

Plus code cleanup in additional commit.

https://issues.apache.org/jira/browse/HDDS-2522

## How was this patch tested?

`TestSecureOzoneCluster` now passes .  No other code changed.

### CODE REVIEW

**Comment by adoroszlai** [comment]
```
[INFO] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 39.348 s - in org.apache.hadoop.ozone.TestSecureOzoneCluster
```

https://github.com/elek/ozone-ci-03/blob/7e4ac617e97e438cfb97d6fd7f28b1f4ae9f4858/pr/pr-hdds-2522-j49tb/integration/output.log#L2301

**Comment by xiaoyuyao** [comment]
Change LGTM, +1 thanks @adoroszlai  for fixing this.

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java [hunk 1]
```diff
@@ -55,10 +53,8 @@
 import org.apache.hadoop.metrics2.lib.DefaultMetricsSystem;
 import org.apache.hadoop.minikdc.MiniKdc;
 import org.apache.hadoop.net.NetUtils;
-import org.apache.hadoop.net.ServerSocketUtil;
 import org.apache.hadoop.ozone.client.CertificateClientTestImpl;
 import org.apache.hadoop.ozone.common.Storage;
-import org.apache.hadoop.ozone.om.OMConfigKeys;
```
Comment (dineshchitlangia) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java]: I think this import should not be removed. This seems to be causing compilation failure:
_[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[178,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[179,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[181,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[183,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster
[ERROR] /github/workspace/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java:[185,16] cannot find symbol
  symbol:   variable OMConfigKeys
  location: class org.apache.hadoop.ozone.TestSecureOzoneCluster_

Comment (adoroszlai) [reply hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java]: Thanks @dineshchitlangia, just noticed and fixed it.

A new usage of `OMConfigKeys` was added in the second merge from master.  I usually rebuild locally, but it seems I forgot it in this case.  Thankfully CI caught it.

In the end, I had to remove the new code, though, as it introduced failures in token-related test cases.

**Comment by adoroszlai** [comment]
Thanks @xiaoyuyao and @dineshchitlangia for reviewing this.

**Pull request merged**